### PR TITLE
Headers

### PR DESCRIPTION
**Pull request by sherrynance**
Made all headers consistent across all of the pages with article and p.

Adjusted the font size of the heading.

Removed the hr from all but the home page.

### CODE REVIEW

File: html/find-help.html [hunk 1]
```diff
@@ -61,10 +61,12 @@
     <section class="container-fluid">
       <div class="row justify-content-center header">
         <div class="col-12">
-          <h1 class="text-center">Real People Want To Help</h1>
-          <p class="text-center">You don’t need to stay silent about being hungry any longer. There are people around
-            you who understand and are waiting to hear from you. They want to help.</p>
-
+          <h1 class="text-center">REAL PEOPLE WANT TO HELP</h1>
+          <article class="p-3">
+          <p>You don’t need to stay silent about being hungry any longer.</p>
```
Comment (amymcgowan) [inline html/find-help.html]: let's indent these p tags

File: css/main.css [hunk 2]
```diff
@@ -6,8 +6,8 @@
     --dk-gray: #3F4443;
 }
 
-.intro {
-    font-size: 20px;
+.intro h2 {
+
```
Comment (amymcgowan) [inline css/main.css]: is this an empty line?  maybe let's take it out?

File: html/help-others.html [hunk 3]
```diff
@@ -61,14 +61,12 @@
...
-            <li>There are hungry students on your campus and you’ve probably unknowingly met many of them.</li>
-            <li>Hunger is often a “silent” issue on campus because of fear or shame to talk about it.</li>
-            <li>Even some students on meal plans don’t get enough to eat.</li>
-          </ul>
+          <h1 class="text-center">YOU CAN MAKE A DIFFERENCE</h1>
+          <article class="p-3">
+            <p>There are hungry students on your campus and you’ve probably unknowingly met many of them.</p>
+            <p>Hunger is often a “silent” issue on campus because of fear or shame to talk about it.</p>
+            <p>Even some students on meal plans don’t get enough to eat.</p>
+            </article>
```
Comment (amymcgowan) [inline html/help-others.html]: un-indent this one level please :)

Comment (amymcgowan) [inline html/help-others.html]: un-indent this one level, please :)

File: html/find-help.html (hunk 1 above)
Comment (sherrynance) [reply html/find-help.html]: done

File: css/main.css (hunk 2 above)
Comment (sherrynance) [reply css/main.css]: done

File: html/help-others.html (hunk 3 above)
Comment (sherrynance) [reply html/help-others.html]: done

Comment (sherrynance) [reply html/help-others.html]: done

**Comment by amymcgowan** [comment]
Thanks!

**Pull request merged**